app/uk/gov/hmrc/serviceconfigs/persistence/FrontendRouteRepository.scala

```diff
@@ -123,16 +123,15 @@ class FrontendRouteRepository @Inject()(
       .toFuture()
       .map(_.wasAcknowledged())
 
-  def replaceAll(environment: String, routes: Set[MongoFrontendRoute]): Future[Unit] = {
+  def replaceAll(environment: String, routes: Set[MongoFrontendRoute]): Future[Unit] =
```

Comment:
should we call this `set` since you are providing an `environment` filter?

Reply:
Thanks changed to replaceEnv as agreed